.contrib/Parser/DATAS/06 - Expansion Features/06 Legion/Legion Class Halls/Mardum The Shattered Abyss (Demon Hunter).lua

```diff
@@ -182,6 +182,7 @@ _.ExpansionFeatures =
 							["g"] = {
 								follower(718),	-- Champion: Akama
 							},
+							["description"] = "This follower is only available if you sided with |cFFFFD700Altruis the Sufferer|r.",
```

Comment:
We typically like the `["g"]` field to be the last field within a group since it can contain a lot of data in some cases.

Reply:
Fixed it, I wish I was more familiar with coding and the structure of the addon.